src/test-container.ts

```diff
@@ -63,6 +63,12 @@ export interface StopOptions {
   removeVolumes: boolean;
 }
 
+export interface ExecOptions {
+  Tty: boolean;
```

Comment:
let's use camelCase for any publicly exposed definitions. You'll also need to update the `exec` method in the `StartedTestContainer` interface below

Reply:
the idea was to keep the same case as the Dockerode specification. 

interface ExecOptions and type ExecContainerOptions now have camelCase attributs. StartedTestContainer updated accordingly

Reply:
you'll also need to update the exec method in the StartedTestContainer to add the new options

Reply:
I included  options? in the exec method signature in StartedTestContainer